compiler/rustc_resolve/src/build_reduced_graph.rs

```diff
@@ -1304,20 +1304,11 @@ impl<'a, 'b> Visitor<'b> for BuildReducedGraphVisitor<'a, 'b> {
     fn visit_item(&mut self, item: &'b Item) {
         let orig_module_scope = self.parent_scope.module;
         self.parent_scope.macro_rules = match item.kind {
-            ItemKind::MacroDef(..) => {
-                let macro_rules_scope = self.define_macro(item);
-                visit::walk_item(self, item);
-                macro_rules_scope
```

Comment:
Err hmm it may have been intentional that `parent_scope.macro_rules` wasn't set until after `walk_item`. This change is invalid then, sorry for the noise.